var/spack/repos/builtin/packages/cubist/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+import os
+
+
+class Cubist(Package):
+    """Cubist is a powerful tool for generating rule-based models that
+    balance the need for accurate prediction against the requirements of
+    intelligibility. Cubist models generally give better results than those
+    produced by simple techniques such as multivariate linear regression,
+    while also being easier to understand than neural networks."""
+
+    homepage = "https://www.rulequest.com"
+    url      = "https://www.rulequest.com/GPL/Cubist.tgz"
+
+    version('2.07', 'f2b20807cd3275e775c42263a4efd3f50df6e495a8b6dc8989ea2d41b973ac1a')
```

Comment:
The URL provided will always download the latest version, This hash will fail if the app gets updated.

Reply:
It's the only link provided though their download page: https://www.rulequest.com/download.html

Reply:
Unfortunately, we have many packages like this. For example, the `triangle` package has the same problem. If you look at https://www.rulequest.com/GPL/, Cubist hasn't been updated in almost a decade, so I don't think we'll have to worry too much about frequent hash changes.